.github/workflows/test.yml

```diff
@@ -79,7 +79,7 @@ jobs:
       with:
         node-version: ${{ matrix.node }}
     - name: Update npm to a version that supports workspaces (v7 or later)
-      if: ${{ matrix.node < 16 }}
+      if: ${{ startsWith(matrix.node, '14') }}
```

Comment:
Before this change the run with `matrix.node` equal to `'14.18.0'` was *not* running this step.